app/controllers/conversations_controller.rb

```diff
@@ -117,9 +117,9 @@ def new
 
   def contacts_data
     current_user.contacts.mutual.joins(person: :profile)
-      .pluck(*%w(contacts.id profiles.first_name profiles.last_name people.diaspora_handle))
-      .map {|contact_id, *name_attrs|
-        {value: contact_id, name: ERB::Util.h(Person.name_from_attrs(*name_attrs)) }
-      }
+                .pluck(*%w(contacts.id profiles.full_name people.diaspora_handle))
```

Comment:
- Pass array contents as separate arguments.
- `%w`-literals should be delimited by `[` and `]`.